lnrpc/routerrpc/router.proto

```diff
@@ -16,7 +34,7 @@ service Router {
     */
     rpc SendPaymentV2 (SendPaymentRequest) returns (stream lnrpc.Payment);
```

Comment:
should we also add this and all others? I'm assuming they are not added here because they are not referenced in the API documentation tool?

Reply:
Correct. It's possible there is an existing incantation of lncli which uses this RPC but it wasn't obvious to me when I stepped through the various command services.